commands/list.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs')
+const tasks = JSON.parse(fs.readFileSync('./tasks.json', 'utf8'))
+
+const list = () => {
+  const longestID = Math.max.apply(null, tasks.map(task => task.id.toString().length))
+
+  if (longestID < 1) {
```

Comment:
This is kindof a roundabout way to test if there are any tasks. A more direct route would be to check if `tasks.length == 0`.